app/controllers/categories_controller.rb

```diff
@@ -1,6 +1,7 @@
 class CategoriesController < ApplicationController
   def show
     @category = Category.find(params[:id])
+    @content_header = "#{@category.name}"
```

Comment:
We could just call @category.name in the view since this is display logic; no need for this to be in the controller